benchmarks/scala-dotty/src/main/scala/org/renaissance/scala/dotty/Dotty.scala

```diff
@@ -24,14 +24,19 @@ import scala.collection.compat._
 @Summary("Runs the Dotty compiler on a set of source code files.")
 @Licenses(Array(License.BSD3))
 @Repetitions(50)
+@Parameter(
+  name = "batch_compilation",
```

Comment:
Minor: might want to consider dashes in parameter names, since it's easier to write.

Reply:
They are not meant to be specified on the command line -- they are mostly written down once for every configuration, so it's relatively infrequent. And we would have to redo all the parameter names :-)